.coveragerc

```diff
@@ -1,2 +1,6 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# SPDX-FileCopyrightText: Copyright 2020 Micron Technology, Inc.
```

Comment:
Having `Copright` feels redundant.

Reply:
FSFe uses `SPDX-FileCopyrightText: 2022 Free Software Foundation Europe e.V. <https://fsfe.org>`

Reply:
I agree it feel redundant, but on the plus side it results in a text string that aligns to https://www.copyright.gov/circs/circ03.pdf, so it complies with REUSE and covers all bases.

Reply:
> Copyright notice is optional for unpublished works, foreign works, or works published on or after March 1, 1989. When notice is optional, copyright owners can use any form of notice they wish. However, works first published after March 1, 1989, may need to comply with statutory formalities to prevent a defendant from invoking an innocent infringement defense in a copyright infringement action.

Either way is fine, but this is buried in that document.

Reply:
Yes, but further down in that same document you'll see a section on why you should include a notice even though it's not strictly required, which is why it's considered a best practice.

Reply:
What we have now is consistent with the SPDX spec and examples: https://spdx.github.io/spdx-spec/v2.3/file-information/#88-copyright-text-field

The REUSE spec says only that "[The SPDX-FileCopyrightText tag MUST be followed by a copyright notice.](https://reuse.software/spec/#comment-headers)"  It does not specify a format for such notices, though it provides an example which is indeed different from what we now have.  However, it seems logical that REUSE accepts any SPDX-FileCopyrightText string that meets the SPDX spec.